FastScripts.glyphsPalette/Contents/Resources/plugin.py

```diff
@@ -20,7 +21,8 @@
     NSLayoutAttributeBottom,
     NSLayoutRelationEqual,
     NSLineBreakByTruncatingTail,
-    NSLayoutConstraintOrientationHorizontal
+	NSLayoutConstraintOrientationHorizontal,
+    NSBezelStyleRecessed,
```

Comment:
Doesn't work for the 2nd version.
Maybe add a condition to use NSShadowlessSquareBezelStyle/NSBezelStyleRecessed for 2/3?